README.markdown

```diff
@@ -468,7 +437,7 @@ Prefer the shortcut versions of type declarations over the full generics syntax.
 **Preferred:**
 ```swift
 var deviceModels: [String]
-var employees: [Int: String]
+var employees: [Int : String]
```

Comment:
Hmm, why `employees: ` but `[Int : String]`?

Not having space before colon, having space after colon seems more consistent to me...